### PR TITLE
fixes/django cms 33 compat

### PR DESCRIPTION
**Pull request by czpython**
* Fixes ``KeyError`` when adding Job opening through wizard in 3.3
* Adds event wizard test

### CODE REVIEW

File: aldryn_events/tests/test_wizards.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            'is_published': True,
+            'app_config': self.app_config.pk,
+        }
+        form = CreateEventForm(
+            data=data,
+            wizard_language='en',
+            wizard_user=self.super_user,
+        )
+
+        print form.errors
```
Comment (mkoistinen) [inline aldryn_events/tests/test_wizards.py]: Oops!

**Comment by mkoistinen** [comment]
LGTM once you remove that print statement =)

File: aldryn_events/tests/test_wizards.py (hunk 1 above)
Comment (czpython) [inline aldryn_events/tests/test_wizards.py]: Oops indeed 👯

**Comment by mkoistinen** [comment]
LGTM

**Pull request merged**